### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ivanhanzlicek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +151,11 @@ Congratulations, if the last command worked, you now have kubernetes running in
      ```
      This is necessary, because the default unit definition also defines a host address, so that it cannot be overridden by the configuration file.
 
+     You might need to restart docker in the lxc container. Run
```
Comment (corneliusweig) [inline README.md]: `might need to` -> `should`. I think this step was simply missing and everybody needs to do this, right?

File: README.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ Lxc is similar to docker but aims more to be an OS container instead of just app
 To use it, install lxd and initialize it using `lxd init`. When prompted, answer the following questions:
 
   - Would you like to use LXD clustering? (yes/no) [default=no]:        
-  - Do you want to configure a new storage pool? (yes/no) [default=yes]: yes -> default location
+  - Do you want to configure a new storage pool? (yes/no) [default=yes]: yes -> **dir**
```
Comment (corneliusweig) [inline README.md]: I think what you mean is that one should use some directory based storage provider, right? Would you mind adding a few more words about this?

File: README.md [hunk 3]
```diff
@@ -204,7 +209,14 @@ Congratulations, if the last command worked, you now have kubernetes running in
    $ mv config.tmp ~/.kube/config
    $ kubectl config use-context k8s-lxc-admin@k8s-lxc
    ```
-   The second line does some magic to merge the admin access credentials into the existing `KUBECONFIG` file.
+   The second line does some magic to merge the admin access credentials into the existing `KUBECONFIG` file. If the last command did not work out make sure that your `~/.kube/config` contains context setup at least like below
```
Comment (corneliusweig) [inline README.md]: I don't think this improves the readme. If the context is there, why would if fail in the first place. And even if this check is positive, there are no instructions how to fix this or how to go on.

I'd remove this hunk.

**Review by corneliusweig — CHANGES_REQUESTED** [review]
Thanks for your improvements. Especially the dir-stuff should help others to not run into problems.

File: README.md (hunk 3 above)
Comment (ivanhanzlicek) [reply README.md]: For me the problem was that the context was configured under the different name, and therefore setting kctl config use context to non-existing context didn't make that much sense. But I will remove that for now and we can reiterate on that

**Pull request merged**